serving/docker/scripts/create_virtual_env.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+# used in the dockerfiles to create virtualenvs per engine
+# currently only intended for use in lmi.Dockerfile, need to refactor this to work for trtllm/neuron if needed
+base_directory=$1
+requirements_file=$2
+
+virtual_env_path="${base_directory}/venv"
```

Comment:
Any particular reason for the "/venv" suffix? This makes the name of all virtual envs created with this script "venv". I noticed when activating it in a shell, since it shows the name in parens in the prompt and it was "(venv) ...".

Reply:
yea i don't think this is actually needed - can remove venv.

I also don't need to source/deactivate - can just call the pip from <venv>/bin/pip